recipe/meta.yaml

```diff
@@ -10,10 +10,11 @@ source:
   url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs{{ version | replace(".", "") }}/ghostscript-{{ version }}.tar.gz
   sha256: 6eaf422f26a81854a230b80fd18aaef7e8d94d661485bd2e97e695b9dce7bf7f
   patches:
+    - config.patch  # [arm64]
```

Comment:
Please use the `gnuconfig` package to copy these as described in the blog post

Reply:
Thanks for pointing out the blog post. I was not aware of it.

Reply:
@isuruf For building on `osx-arm64` the `gnuconfig` package indeed works. Still, the CI is crashing on `osx-64`.
Any idea on how to solve this?